src/app/tests/TestCommandInteraction.cpp

```diff
@@ -46,6 +47,7 @@ static System::Layer gSystemLayer;
 static SecureSessionMgr gSessionManager;
 static Messaging::ExchangeManager gExchangeManager;
 static TransportMgr<Transport::UDP> gTransportManager;
+static secure_channel::MessageCounterManager gMessageCounterManager;
```

Comment:
I have another PR to address this problem #5991, it introduce a CHIP Stack class to initialize all chip components, instead of  manually initialize each component by every app.